### PR TITLE
Run the task every 10 minutes in GitHub Actions

### PR DESCRIPTION
**Pull request by erbridge**
Currently, this runs on Heroku, but it can't be moved into a team without paying for a dyno, which it doesn't need. This hopefully removes the need for Heroku.

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Cache Ruby dependencies
+        uses: actions/cache@v2.1.4
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Install dependencies
+        run: bundle install --frozen --jobs 4 --retry 3
```
Comment (CristinaRO) [inline .github/workflows/run.yml:37]: Bundle keeps reminding us that `--frozen` and a bunch of other options are deprecated and will be removed in the future https://bundler.io/man/bundle-install.1.html#OPTIONS

Could we write this in a way that will be forward-compatible with changes in bundler?

Comment (erbridge) [reply .github/workflows/run.yml:37]: Bah. Tat will tech me for copy and pasteing from other places.

File: .github/workflows/run.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          bundle config set path vendor/bundle
+          bundle config set frozen
+
+      - name: Cache Ruby dependencies
+        uses: actions/cache@v2.1.4
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
```
Comment (CristinaRO) [inline .github/workflows/run.yml:35]: I didn't notice this the first time: is it expected that the value ends in a `-`? Was there some other variable there, or is it just one of those quirky things?

Comment (erbridge) [reply .github/workflows/run.yml:35]: It's basically setting a prefix of keys to restore. Honestly, I forget the details, but I trust my past self to have done it correctly?

Comment (erbridge) [reply .github/workflows/run.yml:35]: Yup: https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#matching-a-cache-key

Comment (CristinaRO) [reply .github/workflows/run.yml:35]: Thank you!

**Review by CristinaRO — APPROVED** [review]

**Pull request merged**